src/classes/UTIL_Describe.cls

```diff
@@ -497,11 +511,25 @@ public class UTIL_Describe {
      * @return String The ID of the Household Account record type as a String.
      */
 	public static String getHhAccRecTypeID() {
-	    String recTypeId = getRecTypesMapByDevName('Account').get('HH_Account');
+        //Get user defined Household record type id
+        String recTypeId = UTIL_CustomSettingsFacade.getSettings().Household_Addresses_RecType__c;
+        //If user defined Household record type does not exist
+        if (recTypeId == NULL
+                || recTypeId == '') {
+            //Use default Household record type id
+            recTypeId = getRecTypesMapByDevName('Account').get('HH_Account');
+            //Set default Household record type as user defined Household record type setting
+            Hierarchy_Settings__c setting = Hierarchy_Settings__c.getInstance();
+            setting.Household_Addresses_RecType__c = recTypeId;
+            upsert setting;
+        }
+
 	    if(String.isBlank(recTypeId) && Test.isRunningTest()) {
-	        if(getRecTypesMapByDevName('Account').values().size() > 1)
-	           return getRecTypesMapByDevName('Account').values()[1];
-	        else return null;
+	        if(getRecTypesMapByDevName('Account').values().size() > 1) {
+                return getRecTypesMapByDevName('Account').values()[1];
+            } else {
+                return null;
+            }
```

Comment:
Not for this release cycle, but I've created a work item for cleaning up the indentations i this class. There are a few methods in here that start out with no indentations after the method declaration which throws the rest of them off. Its been filed under tech debt because it's code that's been there awhile but we should clean up at some point. Again, not for this release though :)